### DISCUSSION TITLE
ST7789 240x240 display cutoff

### DISCUSSION THREAD

**Discussion by Bondoz** [nebhead/PiFire#115] [Q&A]
First this is a great project its just what my pellet stove needed so thank you for doing the work. 

My st7789 240x240 display is cut off is, how can i adjust the display to accomodate the 240x240 display?  I'm assuming its dis[playing as a 240x320.  tia   bondo

![PiFire240](https://user-images.githubusercontent.com/112190563/186942997-ab75f382-be46-4d36-811f-3814532c1862.jpg)

**Comment by nebhead** [comment]
Glad you are enjoying the project!  

Yeah, that particular module you have loaded is using 240x320 as a canvas, and thus is going to be displaying like this when put onto a 240x240 canvas.  Unfortunately, this version of the display would need to be redesigned to accommodate.  The other option would be to use the st7789p module instead which is setup to support 240x240.  It's not as pretty as the 240x320 display module, but it will at least fit the information on the screen.  The other drawback is that this particular module doesn't support buttons or encoder if you wanted that as well.  So you have a few options: 
1. Go with the ST7789p module in the wizard setup. 
2. Hack the ST7789_240x320 module to work with at 240x240 screen.  
3. Buy a different display with 240x320 support.  Either the 9341 or the variant of the ST7789 with 240x320 resolution.
